### PR TITLE
*: Fix memory Leak of Operator. #469

### LINKED ISSUE #469: [bug] Memory Leak of Operator.
**Describe the problem**

**To Reproduce**

1. use `pprof` in operator , and the code in cmd/manager/main.go
```
	go func() {
		ip := "0.0.0.0:6060"
		if err := http.ListenAndServe(ip, nil); err != nil {
			fmt.Printf("start pprof failed on %s\n", ip)
			os.Exit(1)
		}
	}()
```
2. kubectl apply -f xxxx
3. see the alloc objets in `localhost:6060`  , found it is increasing.

**Expected behavior**

**Environment:**

- RadonDB MySQL version: operator

### PR DESCRIPTION
**Pull request by acekingke**
### What type of PR is this?

/bug
### Which issue(s) this PR fixes?

Fixes #469

### What this PR does?

Summary:
1. fix the `log` memleaks.

### CODE REVIEW

File: mysqlcluster/mysqlcluster_test.go [hunk 1]
```diff
@@ -24,16 +24,16 @@ import (
 	"testing"
 
 	. "bou.ke/monkey"
+	mysqlv1alpha1 "github.com/radondb/radondb-mysql-kubernetes/api/v1alpha1"
```
Comment (andyli029) [inline mysqlcluster/mysqlcluster_test.go:27]: Fix it.

File: mysqlcluster/syncer/status.go [hunk 2]
```diff
@@ -27,6 +27,9 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/go-logr/logr"
```
Comment (andyli029) [inline mysqlcluster/syncer/status.go:32]: Fix it

**Comment by acekingke** [comment]
Don't modify `github.com/presslabs/controller-util v0.3.0` to `github.com/presslabs/controller-util v0.4.0` 
Although `github.com/presslabs/controller-util v0.4.0` has fixed the `log` memory leak, It has introduced new ones.

**Review by runkecheng — APPROVED** [review]

**Review by andyli029 — APPROVED** [review]
LGTM

**Pull request merged**